src/main/java/com/google/opensesame/servlets/UserServlet.java

```diff
@@ -55,16 +72,29 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   @Override
   // Send a user to datastore. Update the current information about the user if one exists.
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String userGitHubID = request.getParameter("githubID");
-    if (userGitHubID.trim().isEmpty()) {
-      response.sendError(HttpServletResponse.SC_BAD_REQUEST, "Invalid ID");
-      return;
-    }
-    ArrayList<String> userTags = new ArrayList<>(Arrays.asList(request.getParameterValues("tags")));
+    try {
+      String userGitHubID = request.getParameter("githubID");
+      if (userGitHubID.trim().isEmpty()) {
```

Comment:
`isEmpty()` is Java 11 only. Consider:
```suggestion
      String userGitHubID = request.getParameter("githubID").trim();
      if (userGitHubID == null || userGitHubID.length() == 0) {
```

https://stackoverflow.com/questions/32567208/best-way-to-verify-string-is-empty-or-null